docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Marketplace Product Option feature integration
+last_updated: Jul 28, 2021
+Description: This document describes the process how to integrate the Marketplace Product Option feature into a Spryker project.
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the Marketplace Product Option feature into a Spryker project.
+
+
+## Install feature core
+
+Follow the steps below to install the Marketplace Product Option feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | LINK |
+| --------------- | ------- | ---------- |
+| Spryker Core         | {{page.version}}      | [Spryker Core Feature Integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
+| Marketplace Merchant | {{page.version}}      | [Marketplace Merchant feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-merchant-feature-integration.html) |
+
+
+### 1) Install the required modules using Composer
+
+1) Install the required modules:
+
+```bash
+composer require spryker-feature/marketplace-product-options:"202108.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+|-|-|
+| MerchantProductOption | vendor/spryker/merchant-product-option |
+| MerchantProductOptionDataImport | vendor/spryker/merchant-product-option-data-import |
+| MerchantProductOptionGui | vendor/spryker/merchant-product-option-gui |
+| MerchantProductOptionStorage | vendor/spryker/merchant-product-option-storage |
+| MerchantProductOptionExtension | vendor/spryker/merchant-product-option-extension |
+
+{% endinfo_block %}
+
+
+### 2) Set up the database schema and transfer objects
+
+Adjust the schema definition so that entity changes will trigger events:
```

Comment:
1. Which events?
2.  The name of the file is wrong.
3. Why do we need this schema on the project level?